contributing/tools.md

```diff
@@ -0,0 +1,37 @@
+# Tools that may save you and the team some time
+
+## git-clang-format
```

Comment:
Let's make the header more human-focused.  Suggestion: "Automatically format your Git commit with clang-format"